tests/grading/test_export_bonuspoints.py

```diff
@@ -0,0 +1,214 @@
+from datetime import datetime, timedelta
+from io import StringIO
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from django.core.files.uploadedfile import SimpleUploadedFile
+from django.core.management import call_command
+from django.test import TestCase, tag
+
+from inloop.grading.copypasta import jplag_check
+from inloop.grading.management.commands import tud_export_bonuspoints_csv
+from inloop.grading.models import DetectedPlagiarism
+from inloop.solutions.models import Solution, SolutionFile
+
+from tests.accounts.mixins import SimpleAccountsData
+from tests.grading.mixins import DetectedPlagiarismData, PlagiarismTestData
+from tests.grading.test_jplag import FIBONACCI, TemporaryMediaRootTestCase
+from tests.solutions.mixins import PassedSolutionsData, SimpleTaskData, SolutionsData
+
+
+class BonusPointsTestCase(TestCase):
+    def export_bonuspoints(self, category_name, date):
+        """Conveniently execute the bonus points export management command."""
+        stdout = StringIO()
+        with TemporaryDirectory() as path:
+            output_path = Path(path).joinpath("output")
+            args = [category_name, date.strftime("%Y-%m-%d"), output_path]
+            call_command(tud_export_bonuspoints_csv.Command(), *args, stdout=stdout)
+
+            with open(str(output_path), "r") as f:
+                return stdout.getvalue(), f.readlines(), output_path
+
+
+class BonusPointsExportTest(SolutionsData, BonusPointsTestCase):
+    def test_export_bonus_points(self):
```

Comment:
Note to myself: This test spans many functional units at once:
- `get_ripoff_tasks_for_user`
- `calculate_grades`
- `points_for_completed_tasks`
- `filter_zeroes`
These units should be tested separately.